src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -1189,6 +1189,116 @@ groovyscript.wiki.naturesaura.spawning.entity.value=Sets the entity spawned
 groovyscript.wiki.naturesaura.spawning.removeByEntity=Removes all Altar of Birthing recipes that summon the given entity
 groovyscript.wiki.naturesaura.spawning.removeByName=Removes the Altar of Birthing recipe with the given name
 
+# ProdigyTech
+groovyscript.wiki.prodigytech.atomic_reshaper.title=Atomic Reshaper
+groovyscript.wiki.prodigytech.atomic_reshaper.description=Uses hot air and Primordium to convert items, can have a weighted random based output
+groovyscript.wiki.prodigytech.atomic_reshaper.time.value=The time needed to perform the recipe, in ticks. By default, takes the value from the ProdigyTech's config
+groovyscript.wiki.prodigytech.atomic_reshaper.primordium.value=The amount of Primordium units consumed by this recipe. 1 Primordium equals 100 Primordium units
+groovyscript.wiki.prodigytech.explosion_furnace.title=Explosion Furnace
```

Comment:
minor thing, but could you put an additional linbreak between the registries? i think it improves readability